app/server/build.sh

```diff
@@ -34,6 +34,12 @@ if [[ -f .env ]]; then
   source .env
 fi
 
+# Check APPSMITH_DB_URL environment variable for postgres and run the transform script for postgres databases
+if [[ "${APPSMITH_DB_URI}" == *"postgresql://"* ]]; then
+  # If it contains "postgres", run the transform script
+  python3 ./tx/transform.py
+fi
+
```

Comment:
This is a runtime env variable, and shouldn't influence build. Can we instead change the condition to check if the `transform.py` exists?

```suggestion
if [[ -f tx/transform.py ]]; then
  python3 tx/transform.py
fi

```